docs/design/dedicated-volume.md

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
let's separate design doc into another PR.

Reply:
#545

Reply:
ok，why still here？

Reply:
I was adding api refer to that. Now, i can remove it.